schemes/musica/tuvx/musica_ccpp_tuvx.F90

```diff
@@ -35,40 +41,103 @@ subroutine tuvx_init(errmsg, errcode)
     errmsg = ''
 
     grids => grid_map_t( error )
-    if (has_error_occurred(error, errmsg, errcode)) return
+    if (has_error_occurred( error, errmsg, errcode )) return
 
-    profiles => profile_map_t( error )
-    if (has_error_occurred(error, errmsg, errcode)) return
-
-    radiators =>radiator_map_t( error )
-    if (has_error_occurred(error, errmsg, errcode)) return
-
-    tuvx => tuvx_t( filename_of_tuvx_configuration, error )
-    if (has_error_occurred(error, errmsg, errcode)) return
+    height_grid => create_height_grid( vertical_layer_dimension, &
+        vertical_interface_dimension, errmsg, errcode )
+    if (errcode /= 0) return
```

Comment:
I think the error checking function call can be consistent unless there is reason to not to
```suggestion
    if (has_error_occurred( error, errmsg, errcode )) return
```